universe/api/dto/dto.go

```diff
@@ -28,6 +30,13 @@ type SpaceEffectiveOptions SpaceOptions
 
 type SpaceEffectiveSubOptions map[string]any
 
+type ExploreOption struct {
+	ID          uuid.UUID  `json:"id,omitempty"`
```

Comment:
I'm not sure about omitempty, because I don't know if it works for arrays

Reply:
i think it did work, since it left out parameters from response if they don't exist